AutocompleteClientTests/FW/Logic/Worker/ConstructorIOQuizIntegrationTests.swift

```diff
@@ -241,29 +244,39 @@ class ConstructorIOQuizIntegrationTests: XCTestCase {
             let responseData = response.data!
 
             XCTAssertNil(cioError)
-            XCTAssertNotNil(responseData.versionId)
-            XCTAssertNotNil(responseData.result)
-            XCTAssertNotNil(responseData.result.filterExpressions)
-            XCTAssertTrue(responseData.result.resultsUrl.contains("https://ac.cnstrc.com/browse/items?key=key_vM4GkLckwiuxwyRA&num_results_per_page=10&collection_filter_expression=%7B%22and%22%3A%5B%7B%22name%22%3A%22group_id%22%2C%22value%22%3A%22BrandX%22%7D%2C%7B%22or%22%3A%5B%7B%22name%22%3A%22Color%22%2C%22value%22%3A%22Blue%22%7D%2C%7B%22name%22%3A%22Color%22%2C%22value%22%3A%22red%22%7D%5D%7D%5D%7D"))
+            XCTAssertNotEqual(responseData.versionId, "")
+            XCTAssertNotEqual(responseData.sessionId, "")
```

Comment:
What's the difference between using `XCTAssertNotEqual` vs `XCTAssertNotNil`? 🤔

Reply:
![Screenshot 2023-04-17 at 8 06 22 PM](https://user-images.githubusercontent.com/57369888/232937682-3866a94e-06b3-4b12-9bae-adefce50f4f9.png)

We set it to "" if it doesn't exist. So the test would actually always pass before.

Reply:
Ooo, I wonder if the places where we use `XCTAssertNotNil` isn't being used correctly 😅 

Also, I think you posted a blank white image? 🤔

Reply:
🤔 Might need to go through and make it consistent. I think for some we set it to nil, some we set it to "".